src/azure-cli/azure/cli/command_modules/privatedns/tests/latest/test_privatedns_commands.py

```diff
@@ -722,12 +722,10 @@ def test_PatchRecordSet_SoaRecord_ExpectRecordSetUpdated(self, resource_group):
         self.kwargs['serialNumber'] = 5
         self.kwargs['host'] = 'clitest.hostmaster.com'
         self.cmd('az network private-dns record-set soa update -g {rg} -z {zone} \
-            -e {email} -x {expireTime} -t {host} -m {minimumTtl} -f {refreshTime} -r {retryTime} -s {serialNumber}', checks=[
+            -e {email} -x {expireTime} -m {minimumTtl} -f {refreshTime} -r {retryTime} -s {serialNumber}', checks=[
```

Comment:
Is there any reason for you to remove `-t`?

Reply:
Yes I just rerun the test. The `-t` is used to change hostname but we don't allow to update the hostname except in certain cases. So can't really test it with a random hostname. So removed it from the test.